### DISCUSSION TITLE
Does Panku work for HTML 5?

### DISCUSSION THREAD

**Discussion by pewcworrell** [Ark2000/PankuConsole#215] [Q&A]
I have it opening, but the textures aren't loading so the background is black, and none of the expressions nor the native logger seem to work. I'm wondering if there's some setting I need to change or if its just expected that Panku doesn't work for web?

**Comment by pewcworrell** [comment]
![image](https://github.com/user-attachments/assets/b26cc360-785d-47a7-916a-8f827c72bf02)
I spread out these windows but the 4 on the left open automatically overlapping in the top left. As you can see my global objects aren't found, and the print("Hello") doesn't even show in the native_logger window.

**Comment by worron** [comment]
Unfortunately at the moment panku really doesn't work with html. It's definitely not a matter of settings or your local setup.

**Comment by pewcworrell** [comment]
Actually, I tried again after getting it from your repo directly rather than through Godot's AssetLib and everything worked perfectly! I should note that I also have "Extensions Support" and "Thread Support" enabled.
![image](https://github.com/user-attachments/assets/00096f5a-e86b-4085-858c-6b527e8b0380)

**Reply by worron** [reply]
That's good to hear. Did you enable `Enable File Logging` in your project settings? And it works without any issues after? I was thought the option was disabled by default because some problems with logging outside PC platform, seems I was just out of the loop.

Anyway, using module from repository is always an option. There can be some nice but not released yet improvements and fixes.
